### ISSUE TITLE
error in build win32 32bit

### ISSUE THREAD

**Issue by triasbrata** [wexond/browser-base#258]
hey guys your had good project, and  i got error when  try to run command `npm run compile-win32` but in `package.json` i had litle change to make compile  for 32bit. and when i run it igot error in compile native module. 
my env when run this project is
```
windows 10 64bit
nodejs 12
```

this my `package.json`
```
...
 "scripts": {
    "dev": "cross-env NODE_ENV='dev' node fuse.js",
    "build": "node fuse.js",
    "start": "cross-env ENV='dev' electron .",
    "compile-win32": "npm run build && electron-builder -w -p never --ia32",
    "compile-darwin": "npm run build && electron-builder -m -p always",
    "compile-linux": "npm run build && electron-builder -l -p always",
    "lint": "tslint \"src/**/*.ts*\"",
    "lint-fix": "prettier --write \"src/**/*.ts*\" && tslint \"src/**/*.ts*\" --fix",
    "postinstall": "electron-builder install-app-deps"
  },
...
 "devDependencies": {
    "@types/axios": "0.14.0",
    "@types/chrome": "0.0.86",
    "@types/gsap": "1.20.2",
    "@types/leveldown": "4.0.0",
    "@types/levelup": "3.1.1",
    "@types/nedb": "1.8.8",
    "@types/node": "12.6.8",
    "@types/react": "16.8.23",
    "@types/react-dom": "16.8.4",
    "@types/styled-components": "4.1.18",
    "concurrently": "4.1.1",
    "cross-env": "5.2.0",
    "electron": "6.0.0-beta.15",
    "electron-builder": "21.1.1",
    "fuse-box": "3.7.1",
    "prettier": "1.18.2",
    "terser": "4.1.2",
    "tslint": "5.18.0",
    "tslint-config-airbnb": "5.11.1",
    "tslint-config-prettier": "1.18.0",
    "typescript": "3.5.3",
    "typescript-plugin-styled-components": "1.4.3"
  },
  "dependencies": {
    "@cliqz/adblocker": "0.11.0",
    "axios": "0.19.0",
    "bookmarks-parser": "1.5.0",
    "electron-extensions": "2.0.0",
    "electron-log": "3.0.6",
    "electron-updater": "4.1.2",
    "extract-file-icon": "0.3.1",
    "file-type": "12.0.1",
    "gsap": "2.1.3",
    "icojs": "0.13.1",
    "mobx": "5.13.0",
    "mobx-react": "6.1.1",
    "mouse-hooks": "0.4.1",
    "nedb": "1.8.0",
    "node-vibrant": "3.2.0-alpha",
    "node-window-manager": "1.0.3",
    "pretty-bytes": "5.2.0",
    "react": "16.8.6",
    "react-dom": "16.8.6",
    "react-windows-controls": "1.0.5",
    "styled-components": "4.3.2"
  },
  "cmake-js": {
    "runtime": "electron",
    "runtimeVersion": "5.0.4"
  }
```
and this the error
```
 writing effective config  file=dist\builder-effective-config.yaml
  • rebuilding native dependencies  dependencies=extract-file-icon@0.3.1, leveldown@5.1.1, mouse-hooks@0.4.1, node-window-manager@1.0.3 platform=win32 arch=ia32
  ⨯ cannot execute  cause=exit status 1
                    out=
    > leveldown@5.1.1 install F:\ThorDev\wexond\node_modules\leveldown
    > node-gyp-build


    > extract-file-icon@0.3.1 install F:\ThorDev\wexond\node_modules\extract-file-icon
    > npm run rebuild


    > extract-file-icon@0.3.1 rebuild F:\ThorDev\wexond\node_modules\extract-file-icon
    > node-gyp rebuild


    F:\ThorDev\wexond\node_modules\extract-file-icon>if not defined npm_config_node_gyp (node "C:\Users\IQLand\AppData\Roaming\nvm\v12.0.0\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Users\IQLand\AppData\Roaming\nvm\v12.0.0\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
    Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
      addon.cc
      win_delay_load_hook.cc
         Creating library F:\ThorDev\wexond\node_modules\extract-file-icon\build\Release\addon.lib and object F:\ThorDev\wexond\node_modules\extract-file-icon\build\Release\addon.exp
      addon.vcxproj -> F:\ThorDev\wexond\node_modules\extract-file-icon\build\Release\\addon.node

    > mouse-hooks@0.4.1 install F:\ThorDev\wexond\node_modules\mouse-hooks
    > npm run rebuild


    > mouse-hooks@0.4.1 rebuild F:\ThorDev\wexond\node_modules\mouse-hooks
    > node-gyp rebuild


    F:\ThorDev\wexond\node_modules\mouse-hooks>if not defined npm_config_node_gyp (node "C:\Users\IQLand\AppData\Roaming\nvm\v12.0.0\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Users\IQLand\AppData\Roaming\nvm\v12.0.0\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
    Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
      windows.cc
      win_delay_load_hook.cc
         Creating library F:\ThorDev\wexond\node_modules\mouse-hooks\build\Release\addon.lib and object F:\ThorDev\wexond\node_modules\mouse-hooks\build\Release\addon.exp
      addon.vcxproj -> F:\ThorDev\wexond\node_modules\mouse-hooks\build\Release\\addon.node

    > node-window-manager@1.0.3 install F:\ThorDev\wexond\node_modules\node-window-manager
    > npm run rebuild


    > node-window-manager@1.0.3 rebuild F:\ThorDev\wexond\node_modules\node-window-manager
    > node-gyp rebuild


    F:\ThorDev\wexond\node_modules\node-window-manager>if not defined npm_config_node_gyp (node "C:\Users\IQLand\AppData\Roaming\nvm\v12.0.0\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Users\IQLand\AppData\Roaming\nvm\v12.0.0\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
    Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
      windows.cc
      win_delay_load_hook.cc
    f:\thordev\wexond\node_modules\node-window-manager\lib\windows.cc(89): warning C4018: '<': signed/unsigned mismatch [F:\ThorDev\wexond\node_modules\node-window-manager\build\addon.vcxproj]
    f:\thordev\wexond\node_modules\node-window-manager\lib\windows.cc(96): error C2666: 'Napi::Object::operator []': 3 overloads have similar conversions [F:\ThorDev\wexond\node_modules\node-window-manager\build\addon.vcxproj]
      f:\thordev\wexond\node_modules\node-addon-api\napi-inl.h(868): note: could be 'Napi::Value Napi::Object::operator [](uint32_t) const' (compiling source file ..\lib\windows.cc)
      f:\thordev\wexond\node_modules\node-addon-api\napi-inl.h(864): note: or       'Napi::Value Napi::Object::operator [](const std::string &) const' (compiling source file ..\lib\windows.cc)
      f:\thordev\wexond\node_modules\node-addon-api\napi-inl.h(860): note: or       'Napi::Value Napi::Object::operator [](const char *) const' (compiling source file ..\lib\windows.cc)
      f:\thordev\wexond\node_modules\node-addon-api\napi-inl.h(856): note: or       'Napi::Object::PropertyLValue<uint32_t> Napi::Object::operator [](uint32_t)' (compiling source file ..\lib\windows.cc)
      f:\thordev\wexond\node_modules\node-addon-api\napi-inl.h(852): note: or       'Napi::Object::PropertyLValue<std::string> Napi::Object::operator [](const std::string &)' (compiling source file ..\lib\windows.cc)
      f:\thordev\wexond\node_modules\node-addon-api\napi-inl.h(848): note: or       'Napi::Object::PropertyLValue<std::string> Napi::Object::operator [](const char *)' (compiling source file ..\lib\windows.cc)
      f:\thordev\wexond\node_modules\node-window-manager\lib\windows.cc(96): note: or       'built-in C++ operator[(napi_value, int)'
      f:\thordev\wexond\node_modules\node-window-manager\lib\windows.cc(96): note: while trying to match the argument list '(Napi::Array, int)'

                    errorOut=gyp ERR! build error
    gyp ERR! stack Error: `C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild.exe` failed with exit code: 1
    gyp ERR! stack     at ChildProcess.onExit (C:\Users\IQLand\AppData\Roaming\nvm\v12.0.0\node_modules\npm\node_modules\node-gyp\lib\build.js:262:23)
    gyp ERR! stack     at ChildProcess.emit (events.js:196:13)
    gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:256:12)
    gyp ERR! System Windows_NT 10.0.17134
    gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\IQLand\\AppData\\Roaming\\nvm\\v12.0.0\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
    gyp ERR! cwd F:\ThorDev\wexond\node_modules\node-window-manager
    gyp ERR! node -v v12.0.0
    gyp ERR! node-gyp -v v3.8.0
    gyp ERR! not ok
    npm ERR! code ELIFECYCLE
    npm ERR! errno 1
    npm ERR! node-window-manager@1.0.3 rebuild: `node-gyp rebuild`
    npm ERR! Exit status 1
    npm ERR!
    npm ERR! Failed at the node-window-manager@1.0.3 rebuild script.
    npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

    npm ERR! A complete log of this run can be found in:
    npm ERR!     C:\Users\IQLand\AppData\Roaming\npm-cache\_logs\2019-07-26T07_32_18_185Z-debug.log
    npm ERR! code ELIFECYCLE
    npm ERR! errno 1
    npm ERR! node-window-manager@1.0.3 install: `npm run rebuild`
    npm ERR! Exit status 1
    npm ERR!
    npm ERR! Failed at the node-window-manager@1.0.3 install script.
    npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

    npm ERR! A complete log of this run can be found in:
    npm ERR!     C:\Users\IQLand\AppData\Roaming\npm-cache\_logs\2019-07-26T07_32_18_218Z-debug.log

                    command='C:\Program Files\nodejs\node.exe' 'C:\Users\IQLand\AppData\Roaming\nvm\v12.0.0\node_modules\npm\bin\npm-cli.js' rebuild extract-file-icon@0.3.1 leveldown@5.1.1 mouse-hooks@0.4.1 node-window-manager@1.0.3                    command='C:\Program Files\nodejs\node.exe' 'C:\Users\IQLand\AppData\Roaming\nvm\v12.0.0\node_modules\npm\bin\npm-cli.js' rebuild extract-file-icon@0.3.1 leveldown@5.1.1 mouse-hooks@0.4.1 node-window-manager@1.0.3
                    workingDir=
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wexond@2.2.3 compile-win32: `npm run build && electron-builder -w -p never --ia32`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the wexond@2.2.3 compile-win32 script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\IQLand\AppData\Roaming\npm-cache\_logs\2019-07-26T07_32_18_294Z-debug.log
```

**Comment by triasbrata** [comment]
sorry for my bad engllish

**Comment by sentialx** [comment]
Fixed in 2.2.4
